amqp/1.7/modules/ROOT/pages/amqp-mtls.adoc

```diff
@@ -0,0 +1,28 @@
+= Handle Message Acknowledgment - Mule 4
+:page-aliases: connectors::amqp/amqp-mtls.adoc
+
+Anypoint Connector for AMQP provides mTLS connection capabilities.
+
+== Connection setup
+
+In order to connect to an AMQP broker with mutual TLS (mTLS) authenthication enabled you need the following items:
+* Client Truststore including the Certificate Authority's certificates required for server validation.
+* Client Keystore providing the client private key and certificate required to authentication.
+
+The options useTls and useSasl must be enabled.
+
+`[source,xml,linenums]
+----
+<amqp:connection host="localhost" port="${rabbitmq.tls.port}" useTls="true" useSasl="true" >
+            <tls:context >
+                <tls:trust-store path="mtls/truststore.jks" password="test" type="jks" />
+                <tls:key-store type="jks" path="mtls/test.jks" alias="test" keyPassword="test" password="test" />
+            </tls:context>
+</amqp:connection>
+----
+`
+== See Also
```

Comment:
Wouldl be good to link to this doc https://docs.mulesoft.com/mule-runtime/4.3/tls-configuration